internal/scraper/scraper.go

```diff
@@ -97,21 +96,7 @@ func (s *Scraper) Scrape(url string, files services.FilesService) (models.Recipe
 }
 
 func (s *Scraper) fetchDocument(url string) (*goquery.Document, error) {
-	req, err := http.NewRequest(http.MethodGet, url, nil)
-	if err != nil {
-		return nil, err
-	}
-
-	const mozilla = "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:122.0) Gecko/20100101 Firefox/122.0"
-
-	host := getHost(url)
-	switch host {
-	case "aberlehome", "bettybossi", "marmiton", "puurgezond", "reddit", "thepalatablelife":
-		req.Header.Set("User-Agent", mozilla)
-	case "ah":
-		req.Header.Set("Accept-Language", "nl")
-		req.Header.Set("User-Agent", mozilla)
-	}
+	req, err := services.GetGetRequestForUrl(url)
```

Comment:
Wow, my bad! I forgot to handle the error at line 100. Could you please add the following statement after line 99?
```go
if err != nil {
    return nil, err
}
```